docs/resources/bgp_peer.md

```diff
@@ -0,0 +1,54 @@
+# mikrotik_pool
+
+Creates a Mikrotik [BGP Peer](https://wiki.mikrotik.com/wiki/Manual:Routing/BGP#Peer).
+
+## Example Usage
+
+```hcl
+resource "mikrotik_bgp_peer" "peer" {
+  name = "bgp-peer-name"
+  remote_as = 65533
+  remote_address = "172.21.16.20"
+  peer = "name of bgp peer" # mikrotik_bgp_peer.name
```

Comment:
Shouldn't this be `instance`? I don't see a `peer` argument for this.

Also I think this example should include a `mikrotik_bpg_instance` rather than a made up instance name. That gives people a better understanding of how to use this.